doc/index.rst

```diff
@@ -227,63 +232,59 @@ It can be a regularly scheduled duration (quarterly, monthly, weekly, etc) or ba
 For example, after some number of features are complete or simply based on a schedule of milestones.
 Whatever form this takes, it will be defined on the High-level lsst.io site for each application.
 
-Gitflow Workflow
+Our Git Workflow
```

Comment:
I would suggest something a bit more formal, e.g.; "Telescope and Site Git Workflow"